src/test/resources/features/04_employment.feature

```diff
@@ -209,4 +210,74 @@ Feature: Employment Journeys - Income Tax Submission
+      |Enrolment Key 2    |HMRC-AS-AGENT                                                                      |
+      |Identifier Name 2  |AgentReferenceNumber                                                               |
+      |Identifier Value 2 |XARN1234567                                                                        |
+    Then the user should be redirected to the "Update and submit an Income Tax Return" page
+    When the user clicks the continue button
+    Then the user should be redirected to the "Your client’s Income Tax Return" page
+    When the user clicks on the employment link
+    Then the user should be redirected to the "Employment" page
+    When the user clicks on the 1st change link
+    Then the user should be redirected to the "Employment details and benefits" page
+    When the user clicks on the employment details link
+    Then the user should be redirected to the "Check your client’s employment details" page
+    When the user clicks on the change Employer link
+    Then the user should be redirected to the "What’s the name of your client’s employer?" page
+    When the user selects the Employer name field and enters a value of Mishima Zaibatsu
+    And the user clicks the continue button
+    Then the user should be redirected to the "Check your client’s employment details" page
+    When the user clicks on the change start date link
+    Then the user should be redirected to the "When did your client start working for their employer?" page
+    When the user selects the Day field and enters a value of 14
+    And the user selects the Month field and enters a value of 07
+    And the user selects the Year field and enters a value of 2012
+    And the user clicks the continue button
+    Then the user should be redirected to the "Check your client’s employment details" page
+    When the user clicks on the change paye reference link
+    Then the user should be redirected to the "What’s the PAYE reference of your client’s employer?" page
+    When the user selects the PAYE reference field and enters a value of 777/7777
+    And the user clicks the continue button
+    Then the user should be redirected to the "Check your client’s employment details" page
+    When the user clicks on the change pay received link
+    Then the user should be redirected to the "How much did your client’s employer pay them?" page
+    When the user selects the Pay field and enters a value of 9000.01
+    And the user clicks the continue button
+    Then the user should be redirected to the "Check your client’s employment details" page
+    When the user clicks on the change tax taken from pay link
+    Then the user should be redirected to the "How much UK tax was taken from your client’s earnings?" page
+    When the user selects the Tax field and enters a value of 720.08
+    And the user clicks the continue button
+    Then the user should be redirected to the "Check your client’s employment details" page
```

Comment:
I think the flow should include the save and continue button on "Check your client’s employment details" page and it's redirect after line 260 like with interest and dividends CYA changes